src/components/Filters/CollapsibleFilter/CollapsibleFilter.js

```diff
@@ -0,0 +1,55 @@
+import './CollapsibleFilter.less';
+import getIcon from '../../iconMap';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+
+const CollapsibleFilter = ({
+  hasChildren,
+  className = '',
+  title,
+  desc,
+  children,
+}) => {
+  const [isOpen, setOpen] = useState(hasChildren);
```

Comment:
I took another look at this, this really should just be 

const [isOpen, setOpen] = useState(true);

and we can remove hasChildren from line 7.  

Looks like all of these filters are open by default, and it really has nothing to do with whether any children exist.

Reply:
I've updated the code to not include hasChildren. This required updates to other components and their unit tests to ensure that everything continues to work as expected